include/swift/AST/NameLookupRequests.h

```diff
@@ -273,6 +274,27 @@ class GenericParamListRequest :
   void cacheResult(GenericParamList *value) const;
 };
 
+class GenericSignatureRequest :
```

Comment:
This should be in TypeCheckRequests since the implementation of evaluate() will use the functions in TypeCheckGeneric.cpp.

Reply:
The generic signature is computed by the verifier so clients not linking Sema all fail.